Contest_API.md

```diff
@@ -1695,10 +1695,10 @@ Properties of clarification message objects:
 | Name           | Type    | Description
 | :------------- | :------ | :----------
 | id             | ID      | Identifier of the clarification.
-| from\_team\_id | ID ?    | Identifier of [ team](#teams) sending this clarification request, `null` if a clarification sent by jury.
-| to\_team\_id   | ID ?    | Identifier of the [ team](#teams) receiving this reply, `null` if a reply to all teams or a request sent by a team.
+| from\_team\_id | ID ?    | Identifier of [team](#teams) sending this clarification request, `null` iff a clarification sent by jury.
+| to\_team\_id   | ID ?    | Identifier of the [team](#teams) receiving this reply, `null` iff a reply to all teams or a request sent by a team.
 | reply\_to\_id  | ID ?    | Identifier of clarification this is in response to, otherwise `null`.
-| problem\_id    | ID ?    | Identifier of associated [ problem](#problems), `null` if not associated to a problem.
+| problem\_id    | ID ?    | Identifier of associated [problem](#problems), `null` iff not associated to a problem.
```

Comment:
Make it explicit that these attributes can *only* be null under the specified conditions.